### PR TITLE
feat: Adds drag and drop for images/files in the agent input box

### LINKED ISSUE #1219: Enhancement: Allow drag and drop, copy & paste image/files in the agent input box.
- Allow drag and drop files on the input box
- Allow copy & paste image

### PR DESCRIPTION
**Pull request by iamsivin**
Fixes #1219 

https://user-images.githubusercontent.com/64252451/123906807-779a8500-d992-11eb-8a3c-f6d61aae24e4.mov

### CODE REVIEW

**Comment by pranavrajs** [comment]
Nice work! 🎊 

One thing I noticed is that `drag and drop message` won't be hidden if I move the cursor away from the editor. Please see attached screencast.

https://user-images.githubusercontent.com/2246121/123900202-43b96280-d986-11eb-9dd4-dac21548ab36.mp4

File: app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue [hunk 1]
```diff
@@ -37,6 +39,17 @@
         :title="$t('CONVERSATION.REPLYBOX.TIP_FORMAT_ICON')"
         @click="toggleFormatMode"
       />
+      <transition name="modal-fade">
+        <div
+          v-show="$refs.upload && $refs.upload.dropActive"
+          class="drag-drop-modal"
+        >
+          <div class="ion-ios-cloud-upload-outline icon"></div>
```
Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue:47]: use `<i>` for icons for the consistency we follow.

**Review by nithindavid — CHANGES_REQUESTED** [review]
Great PR. 🎉

Also can we make the drop zone to entire screen ? So issue mentioned by Pranav would be solved.

File: app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue [hunk 2]
```diff
@@ -198,4 +212,18 @@ export default {
     background: var(--s-100);
   }
 }
+
+.modal-mask {
+  color: var(--s-600);
+  background: rgba(255, 255, 255, 0.9);
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue:218]: Should we move it to a variable?

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**